packages/build/webpack/constants.js

```diff
@@ -23,21 +28,14 @@ if (process.env.LINK) {
     }
 
     labkeyUIComponentsPath = process.env.LABKEY_UI_COMPONENTS_HOME + '/packages/components/src';
+    // lastIndexOf just in case someone is weird and has their LKS deployment under a directory named modules.
+    const lkModulesPath = cwd.slice(0, cwd.lastIndexOf('modules') + 1);
+    freezerManagerPath = lkModulesPath.concat(FREEZER_MANAGER_DIRS).join(path.sep);
+    workflowPath = lkModulesPath.concat(WORKFLOW_DIRS).join(path.sep);
 
-    const freezerManagerRelPath = (lkModuleContainer ? '../../../../../../' : '../../../../../') + 'inventory/packages/freezermanager/src';
-    freezerManagerPath = path.resolve(__dirname, freezerManagerRelPath);
-
-    const workflowRelPath = (lkModuleContainer ? '../../../../../../' : '../../../../../') + 'sampleManagement/packages/workflow/src';
-    workflowPath = path.resolve(__dirname, workflowRelPath);
-}
-if (process.env.npm_package_dependencies__labkey_components) {
-    console.log('Using @labkey/components path: ' + labkeyUIComponentsPath);
-}
-if (process.env.npm_package_dependencies__labkey_freezermanager) {
-    console.log('Using @labkey/freezermanager path: ' + freezerManagerPath);
-}
-if (process.env.npm_package_dependencies__labkey_workflow) {
-    console.log('Using @labkey/workflow path: ' + workflowPath);
+    console.log('Using @labkey/components path:', labkeyUIComponentsPath);
+    console.log('Using @labkey/freezermanager path:', freezerManagerPath);
+    console.log('Using @labkey/workflow path:', workflowPath);
```

Comment:
any specific reason you dropped the `process.env.npm_package_dependencies__` checks? this was added just to prevent console.log noise for those module package.json files which didn't have a given labkey package (i.e. biologics not having freezermanager).

Reply:
I could move it back if you'd like, but I feel like this information is still useful, because we are overriding the path, even if the module isn't present, which is also why I only changed the console.log to be inside of the LINK case, this way you only see it when you're using `start-link`, so the noise is reduced in most cases, except for LINK, which I think is ok.